doc/specs/vulkan/chapters/initialization.txt

```diff
@@ -17,7 +17,7 @@ command:
 include::../protos/vkGetInstanceProcAddr.txt[]
 
   * pname:instance is the instance that the function pointer will be
-    compatible with.
+    compatible with, or `NULL` for commands not dependent on any instance.
```

Comment:
In this line the only point is so the description doesn't lie (if it is "the instance" then it can't be NULL -- so it must be stated too)

>  Currently Valid Usage says that if instance is NULL, pName can only be those three calls and everything else is "undefined"

Precisely, if it is in VU, then the case `instance==NULL` and not one of the three command names is undefined behavior.
I think (and I could be wrong) that the intention is to return NULL in that case too instead.
(Same for the "must not" VU except the oposite is undefined behavior.)